bindings/python/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
-find_package(Python3 ${PYTHON_VERSION} EXACT COMPONENTS Development REQUIRED)
+find_package(Python3 COMPONENTS Development.Module REQUIRED)
```

Comment:
``Development.Module`` is available since CMake 3.18. We should raise ``cmake_minimum_required(VERSION 3.15)``

Reply:
Indeed! Fixed, CI is running :)